build/yaml/samples/nsx_v1alpha1_vpcnetworkconfigurations.yaml

```diff
@@ -12,4 +12,4 @@ spec:
   privateIPv4CIDRs:
     - 172.26.0.0/16
     - 172.36.0.0/16
-  defaultSubnetAccessMode: Private
+  defaultPodSubnetAccessMode: Private
```

Comment:
It's good that we also change other properties to reflect what we changed for the CRD.